pilot/pkg/model/context.go

```diff
@@ -542,8 +542,8 @@ type NodeMetadata struct {
 	// Mostly used when istiod requests the upstream.
 	IstioRevision string `json:"ISTIO_REVISION,omitempty"`
 
-	// Labels specifies the set of workload instance (ex: k8s pod) labels associated with this node. Labels is a
-	// superset of IstioMetaLabels.
+	// Labels specifies the set of workload instance (ex: k8s pod) labels associated with this node.
+	// Note: should not be used during xds generation because pod labels update support.
 	Labels map[string]string `json:"LABELS,omitempty"`
```

Comment:
Maybe we should rename it (the go type, not the `LABELS`), to something like StaticLabels or similar. So its harder to misuse?

Reply:
Will think about it